en/flight_stack/controller_diagrams.md

```diff
@@ -39,6 +39,31 @@ The roll and pitch controllers have the same structure and the longitudinal and
 The yaw controller, however, generates its yaw rate setpoint using the turn coordination constraint in order to minimize lateral acceleration, generated when the aircraft is slipping. 
 The yaw rate controller also helps to counteract adverse yaw effects (https://youtu.be/sNV_SDDxuWk) and to damp the [Dutch roll mode](https://en.wikipedia.org/wiki/Dutch_roll) by providing extra directional damping.
 
+
+## VTOL Flight Controller 
+
+![VTOL Attitude Controller Diagram](../../assets/diagrams/VTOL_controller_diagram.png)
+
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1tVpmFhLosYjAtVI46lfZkxBz_vTNi8VH/view?usp=sharing
+Request access from dev team. -->
+
+This section gives a short overview on the control structure of Vertical Take-off and Landing (VTOL) aircraft. 
+The VTOL flight controller consists of both the multicopter and fixed-wing controllers, either running separately in the corresponding VTOL modes, or together during transitions. 
+The diagram above presents a simplified control diagram. 
+Note the VTOL attitude controller block, which mainly facilitates the necessary switching and blending logic for the different VTOL modes, as well as VTOL-type-specific control actions during transitions (e.g. ramping up the pusher motor of a standard VTOL during forward transition). 
+The inputs into this block are called "virtual" as depending on the current VTOL mode, some are ignored by the controller. 
+
+For a standard and tilt-rotor VTOL, during transition the fixed-wing attitude controller produces the rates setpoint, which are then fed into the separate rate controllers, resulting in torque commands for the multicopter and fixed-wing actuators.
+For tailsitters, during transition the multicopter attitude controller is running. 
+
+The outputs of the VTOL attitude block are separate torque and force commands for the multicopter (actuator_control_0) and fixed-wing (actutor_controls_1) actuators. 
```

Comment:
Typo actutor_controls_1

YOu might want to put these in backticks - ie `actuator_control_0` to make them more visible.